index.html

```diff
@@ -1,39 +1,348 @@
+          <p class="introduction">
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+          <button>Get My Resume</button>
+        </div>
+      </div>
+      <div class="about-line">
+        <hr />
+      </div>
+      <div class="aboutList">
+        <ul>
+          <img src="./images/Rectangle 55.png" alt="" />
+          languages
+          <li>Javascript</li>
+          <li>ruby</li>
+          <li>Html</li>
+          <li>Css</li>
+        </ul>
+        <ul>
+          <img src="./images/Rectangle 56.png" alt="" />
+          Frameworks
+          <li>Bootstrap</li>
+          <li>Ruby on Rails</li>
+          <li>Rspec</li>
+          <li>CapyBara</li>
+          <li>Selenium</li>
+        </ul>
+        <ul>
+          <img src="./images/Rectangle 57.png" alt="" />
+          Skills
+          <li>Codekit</li>
+          <li>Github</li>
+          <li>Codepen</li>
+          <li>Gitlab</li>
+          <li>Terminal</li>
+        </ul>
+      </div>
+    </section>
+    <section>
+      <div class="contact-us">
+        <div class="contact-text">
+          <h1>
+            I'm always interested in hearing about new projects, so if you'd
+            like to chat please get in touch.
+          </h1>
+        </div>
+        <div class="contactUs-form">
+          <form
+            class="contact-form"
+            action="https://formspree.io/f/mzbwnvpb"
+            method="POST"
+          >
+            <input
+              class="input-field"
+              type="text"
+              maxlength="30"
+              required
+              placeholder="Full names"
+            />
+            <input
+              class="input-field"
+              type="email"
+              required
+              placeholder="email address"
+            />
+            <textarea
+              class="text-area"
+              name="message"
+              id=""
+              cols="30"
+              rows="10"
+              required
+            >
+  Enter text header</textarea
+            >
+            <button class="contact-button" type="submit">Get in touch</button>
+          </form>
+        </div>
+      </div>
+    </section>
 
-   
-</body>
-</html>
\ No newline at end of file
+    <footer class="footer">
+      <div class="footer-icons">
+        <ul>
+          <li>
+            <a href=""><img src="./images/vector1.png" alt="" /></a>
+          </li>
+          <li>
+            <a href=""><img src="./images/vector2.png" alt="" /></a>
+          </li>
+          <li>
+            <a href=""><img src="./images/vector3.png" alt="" /></a>
+          </li>
+          <li>
+            <a href=""><img src="./images/vector4.png" alt="" /></a>
+          </li>
```

Comment:
- [ ] Kindly make sure the `space` between these elements match with the `space` in the Figma design.

Your Design
![Screenshot from 2022-12-10 18-06-26 (1)](https://user-images.githubusercontent.com/80458199/206866004-89dbbdf4-a00d-4002-9084-1c5b45a8cd2a.png)


Figma Design

![Screenshot from 2022-12-10 18-48-41](https://user-images.githubusercontent.com/80458199/206865986-582bb958-8965-4228-9754-ba80e4d52cc1.png)

Reply:
Fixed.